templates/mobile/activation/activation.1.schema.json

```diff
@@ -0,0 +1,54 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "title": "activation",
+  "properties": {
+    "identifier": {
+      "type": "string",
+      "description": "hashed version of Google ID"
+    },
+    "client_id": {
+      "type": "string",
+      @COMMON_PATTERN_UUID_1_JSON@,
+      "description": "telemetry client_id, included only if identifer is empty"
+    },
+    "manufacturer": {
+      "type": "string",
+      "description": "Build.MANUFACTURER"
+    },
+    "model": {
+      "type": "string",
+      "description": "Build.MODEL"
+    },
+    "device": {
+      "type": "string"
+    },
+    "distribution_id": {
+      "type": "string"
+    },
+    "locale": {
+      "type": "string",
+      "description": "application locale, such as 'en-US'"
+    },
+    "os": {
+      "type": "string"
+    },
+    "osversion": {
+      "type": "string"
+    },
+    "created": {
+      "type": "string"
+    },
+    "profile_date": {
+      "type": "integer"
+    },
+    "tz": {
+      "type": "string"
+    }
+  },
+  "required": [ "manufacturer", "model", "device", "locale", "os", "osversion" ],
```

Comment:
It's unclear to me which of these fields we should mark as required, so I've generally copied over the list of required fields from the core ping schema.

If one of these required fields is missing, the ping will be rejected in the pipeline and not end up in the dataset.

Reply:
Discussed with :vbaicu and updated this list.